index.html

```diff
@@ -4566,11 +4590,18 @@ <h2 id="AudioListener">
           The AudioListener Interface
         </h2>
         <p>
-          This interface interface is DEPRECATED, as it will be replaced by the
+          This interface represents the position and orientation of the person
```

Comment:
Hard to tell from the diff, but did you really want to get rid of the deprecation?